src/content/docs/en/guides/deploy/zephyr.mdx

```diff
@@ -0,0 +1,145 @@
+
+Zephyr operates on a **Bring Your Own Cloud (BYOC)** model, deploy to your choice of [supported clouds](https://docs.zephyr-cloud.io/cloud) through a unified interface without vendor lock-in. Switch providers anytime without changing your deployment workflow.
+
+## How to deploy
+
+### Automatic Installation
+
+<Steps>
+1. Add the Zephyr integration to your Astro project with the following command. This will install the integration and update your `astro.config.mjs` file automatically:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npx with-zephyr@latest
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm dlx with-zephyr@latest
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn dlx with-zephyr@latest
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+2. Build and deploy your Astro site:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm run build
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm run build
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn run build
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+3. Your application is deployed! Zephyr will provide a deployment URL and comprehensive build analytics.
+</Steps>
+
+### Manual Installation
+
+<Steps>
+1. Install the Zephyr Astro integration:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm install zephyr-astro-integration
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm add zephyr-astro-integration
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn add zephyr-astro-integration
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+2. Add the integration to your `astro.config.mjs`:
+
+    ```js
+    import { defineConfig } from 'astro/config';
+    import { withZephyr } from 'zephyr-astro-integration';
+
+    export default defineConfig({
+      integrations: [
+        withZephyr(),
+      ],
+    });
+    ```
+
+3. Build and deploy your Astro site:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm run build
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm run build
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn run build
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+4. Your application is deployed! Zephyr will provide a deployment URL and comprehensive build analytics.
+</Steps>
+
+### More details
+
+For more detialed information refer to the Zephyr Cloud documentation on [Deploying with Astro](https://docs.zephyr-cloud.io/meta-frameworks/astro).
```

Comment:
```suggestion
For more detialed information refer to the [Zephyr Cloud documentation on deploying with Astro](https://docs.zephyr-cloud.io/meta-frameworks/astro).
```
link text should be as descriptive as possible about both the content and the location the link will take you to! "deploying with Astro" is a bit too general.

Also noting this page doesn't exist (yet)? Is this page expected, and we should wait on merging this until it's live?

Reply:
Yep, waiting to merge this one to be shown on a real URL https://github.com/ZephyrCloudIO/zephyr-documentation/pull/160.

Also lemme tweak this a bit to more descriptive :)

Reply:
I'm going to add my suggestion, since it's descriptive *enough*, and the page is now live in Zephyr docs, and I think that getting this in the Nav component is all that's holding up this merge @Nsttt ! 

If you'd like to make another edit here, or even revisit and improve the guide after it's published, you're always welcome to make a follow up PR, but this doesn't have to hold up this PR.